lesson_06/lesson_06.3/task_02.js

```diff
@@ -0,0 +1,68 @@
+const OPERATION = "+-*/%^";
+const firstNumber = prompt("Please enter first number.");
+const operationSymbol = prompt("Please select operation +, -, *, /, % or ^");
+const secondNumber = prompt("Please enter second number.");
+const isNumber = function checkInput(enterNumber) {
+  if (
+    typeof Number(enterNumber) === "number" &&
+    enterNumber !== null &&
+    !isNaN(enterNumber) &&
+    enterNumber !== ""
+  ) {
+    return true;
+  }
+  return false;
+};
+const isOperation = function checkInputOperation(enterOperation) {
+  if (OPERATION.indexOf(enterOperation) !== -1 && enterOperation !== null) {
+    return true;
+  }
+  return false;
+};
+const isValid = function dateValidation() {
+  if (
+    isNumber(firstNumber) &&
+    isNumber(secondNumber) &&
+    isOperation(operationSymbol)
+  ) {
+    return true;
+  }
+  return false;
+};
+const doMath = function calculateResult(x, znak, y) {
+  let result = 0;
+  if (znak === "+") {
+    result = x + y;
```

Comment:
```suggestion
    return x + y;
```
я б зробив тут і нижче ось так, і змінна для результату тоді не потрібна

Reply:
Виправив.